README.md

```diff
@@ -81,6 +81,13 @@ _Add your project ideas below:_
 -   Mentor(s): Jim Bugwadia (@JimBugwadia)
 -   Issue: Multiple issues - https://github.com/kubernetes-sigs/multi-tenancy/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+MTB
 
+#### Kubernetes Policy Working Group: Policy Violation Custom Resource and Adapters
+
+-   Description: The Kubernetes Policy Working Group is providing an overall architecture that describes both the current policy related implementations as well as future policy related proposals in Kubernetes. As part of this effort, the group is proposing a Kubernetes Custom Resources Definition (CRD) for policy violations and writing adapters to convert from various Kubernetes policy management tools such as OPA, Falco, Kyverno, kube-bench, etc. to report policy violations in a common format. The Policy Violation CRD can then be used by the Dashboard to show violations.
+-   Recommended Skills: Go, Kubernetes
+-   Mentor(s): Jim Bugwadia (@JimBugwadia)
+-   Issue: https://github.com/kubernetes/community/issues/4538
```

Comment:
Why has it been opened under the k/community repo?

Also, any approvers or prior discussion regarding the proposed CRD?

Reply:
I could not find a repo for the `policy` working group under `kubernetes-sigs`. I've asked the WG leads if we can create one, but wanted to proceed with the submission in the meanwhile. Let me know if there is a better way to manage...thanks!

Reply:
@JimBugwadia k/community repo is not a proper place to file the issues like this; @nikhita may provide more details.

But the more important item we have to clarify - who are the approvers of this proposal and has any prior public discussion happened regarding the proposed CRD? If yes, can you please share the links?

Reply:
@idvoretskyi - ok, thanks! 

Where do you suggest we file this issue? The WG page links to this repo:

https://github.com/kubernetes/community/tree/master/wg-policy

The proposal is being discussed in the Policy WG meetings. It is work in progress - not sure what approvals are necessary for that. Can you clarify?

Here are some links:

https://docs.google.com/document/d/1QJWcaJdo8w88tIixiZp9zBMHKghTMrHTyHyk6xcoVhk/edit

https://docs.google.com/document/d/1ihFfEfgViKlUMbY2NKxaJzBkgHh-Phk5hqKTzK-NEEs/edit#

Reply:
@jimbugwadia I'm speaking about the approvers in the Kubernetes community terminology here - https://github.com/kubernetes/community/blob/master/community-membership.md#approver.

I'll pass this proposal to @nikhita for the review and suggestions.

Reply:
@idvoretskyi - thanks for the pointer! 

@nikhita - please let me know what is needed. For other WG submissions, we have used the WG git repo. Do we need to create one for the policy WG? If so, how do we do that?

Reply:
@JimBugwadia Hi! Typically, for GSoC projects, we require that we have a repo in place where students would be making initial contributions and/or their final project ideas. In this case, we'd require a repo under the @kubernetes-sigs org.

Since SIGs can own code (and WGs can't), we'd need related SIGs to "sponsor" this new repo creation. You can find the procedure to request a new repo [here](https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md#sig-repositories). https://github.com/kubernetes/org/issues/528 is a good related example, where repo creation for WG Multitenancy required approval by SIG Auth leads.

Once we have approval from the sponsoring SIG leads, the [GitHub Admin Team](https://github.com/kubernetes/community/tree/master/github-management#github-administration-team) will create the repo, and we can move ahead with this PR. :+1:

Reply:
@JimBugwadia Also, apologies for the additional processes on this! :bowing_woman: 

We want to keep the procedures minimal, but since the Kubernetes project ecosystem is huge, we want to ensure that all project ideas have appropriate approvals and a repo for students to start out with to make it easier for both mentors and mentees once the application period opens.

Reply:
@nikhita - ok, that makes sense. I will work with the WG leads to get the repo created. Thanks!

Reply:
> @JimBugwadia Also, apologies for the additional processes on this! 🙇‍♀
> 
> We want to keep the procedures minimal, but since the Kubernetes project ecosystem is huge, we want to ensure that all project ideas have appropriate approvals and a repo for students to start out with to make it easier for both mentors and mentees once the application period opens.

So if I understand this correctly we could request to create a repo with the "policy-wg" name but its sponsorship has to come from a certain SIG right ?

Reply:
@nikhita So if I understand this correctly we could request to create a repo with the "policy-wg" name but its sponsorship has to come from a certain SIG right ?

Reply:
> @nikhita So if I understand this correctly we could request to create a repo with the "policy-wg" name but its sponsorship has to come from a certain SIG right ?

Yes, that's correct. :+1: